www/src/pages/components/generic-toast.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Generic Toast'
+type: 'component'
+status: 'Stable'
+designStatus: 'TO DO'
+devStatus: 'TO DO'
+notes: |
+  Getting a design update soon
+---
```

Comment:
This file is just a placeholder for the old Toast. Will be removed before merging